into/backends/sparksql.py

```diff
@@ -31,6 +33,19 @@ def iterable_to_sql_context(ctx, seq, **kwargs):
     return append(ctx, append(ctx._sc, seq, **kwargs), **kwargs)
 
 
+@append.register(SQLContext, (JSONLines, Directory(JSONLines)))
+def jsonlines_to_sparksql(ctx, json, dshape=None, name=None, schema=None,
+                          samplingRatio=0.25, **kwargs):
+    # if we're passing in schema, assume that we know what we're doing and
+    # bypass any automated dshape inference
+    if dshape is not None and schema is None:
+        schema = dshape_to_schema(dshape.measure
+                                  if isrecord(dshape.measure) else dshape)
```

Comment:
Ok. I use schema explicitly here because it's an argument that `SQLContext.jsonFile` accepts.